src/hotspot/share/runtime/notificationThread.cpp

```diff
@@ -61,28 +59,11 @@ void NotificationThread::initialize() {
                           vmSymbols::thread_void_signature(),
                           thread_oop,
                           THREAD);
```

Comment:
This was all the code that I thought was unnecessarily duplicated.

Reply:
I don't know why I don't see your reply in the PR anywhere, but I'm glad you're addressing it in JDK-JDK-8269652.